ffv1.md

```diff
@@ -893,7 +908,7 @@ Note, this is different from JPEG-LS, which doesn’t use prediction in run mode
 
 #### Initial Values for the VLC context state
 
-When `keyframe` (see (#frame)) value is 1, all coder state variables are set to their initial state.
+When `keyframe` (see (#frame)) value is 1, all Range coder state variables are set to their initial state.
```

Comment:
The heading talks about VLC not range coding this change doesnt seem to fit

Reply:
thanks. I changed from "Range" to "VLC" here so each use of the term "coder state variables" is contextualized as Range or VLC